projects/canopy/src/lib/icon/docs/guide.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/UI icon/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# UI icon
+
+<p class="standfirst">UI icons are visual depictions of an object, action or idea. They're simple and easy to recognise, and can communicate a lot with very little.</p>
+
+![example](docs/ui-icon/example.png)
+
+## Usage
+
+UI icons should be used to highlight or support key information or give affordance to links and buttons.
+
+Import the icon module into your module:
+
+```js
+@NgModule({
+  ...
+  imports: [ ..., LgIconModule ],
+})
+```
+
+
+
+Import the `LgIconRegistry` service and register your icons inside your module:
+
+```js
+// import the desired icon
+import { lgIconEmail } from '@legal-and-general/canopy';
+
+export class AppModule {
+  // register the icon using the `LgIconRegistry` service
+  constructor(private iconRegistry: LgIconRegistry) {
+    this.iconRegistry.registerIcons([
+      lgIconEmail
+    ]);
+  }
+}
+```
+
+
+
+Your HTML:
+
+```html
+<lg-icon
+  name="email">
+</lg-icon>
+```
+
+## Dos and Don'ts
+
+### Do
+
+![do](docs/ui-icon/do.png)
+
+1. **Do** scale icons along the 4px scale to ensure crisp outlines
+2. **Do** use charcoal or high contrast colour icons on white background
+3. **Do** use tinted colours on light colour backgrounds, ensuring a minimum of 4.51:1 contrast
+4. **Do** use white icons on a high contrast background
+
+### Don't
+
+![dont](docs/ui-icon/dont.png)
+
+1. **Don't** scale icons below 16px
+2. **Don't** scale icons above 64px. Use brand icons or illustration for more prominent graphics
+3. **Don't** overlay icons on photographic backgrounds
+4. **Don't** colour the icon differently to the background e.g. yellow on blue
+5. **Don't** use low-contrast colour combinations
+
+## Additional developer details
+
+### Inputs
+
+| Name   | Description          |  Type  |  Default  | Required |
+| ------ | -------------------- | :----: | :-------: | :------: |
+| <code>name</code> | the name of the icon | string | undefined |   yes    |
```

Comment:
Not a boiggy, but you can use the double backtick (\`\`name\`\`) markdown syntax here instead of `<code/>`

Reply:
Ah, thanks for the tip. The original code had `name` which wasn't rendering correctly so I changed it to <code>name </code>. Good to know that ``name``  works.